components/FormFieldSelect/FormFieldSelect.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react';
+import { Select } from '../Select';
+
+interface FormFieldSelectItem {
+  value: string;
+  label: string;
+}
+
+interface FormFieldSelectProps {
```

Comment:
Make this extend normal `SelectProps`, and pass them into the `Select` component.

Reply:
Updated to extend `SelectProps`